content/en/blog/2025/devex-survey/index.md

```diff
@@ -0,0 +1,148 @@
+---
+
+# Insights from the OpenTelemetry Developer Experience Survey
+
+The OpenTelemetry Developer Experience SIG recently surveyed the community to
+better understand where the SIG could have the most impact on improving the
+developer experience. We received 218 responses, which we will use to guide our
+prioritization of work within the SIG. This post summarizes our findings and
+where we will focus in the near term.
+
+Thank you to all those who participated in the survey\! If you are interested in
+joining us in these efforts or providing additional insights on where you think
+the developer experience of OpenTelemetry can be improved, please don’t hesitate
+to reach out. Contact information for the SIG can be found in the Community
+repo’s list of
+[Cross-Cutting SIGs](https://github.com/open-telemetry/community/?tab=readme-ov-file#cross-cutting-sigs),
+and we have a
+[repository](https://github.com/open-telemetry/sig-developer-experience) of our
+own on GitHub.
+
+## Key Takeaways
+
+The survey responses highlight the need for improved documentation, more
+in-depth examples and more substantial guidance on debugging, configuring and
+deploying OpenTelemetry in Production. Below are the main areas of concern:
+
+- Documentation:
+  - Documentation is difficult to navigate, requiring users to switch between
+    the OpenTelemetry website and GitHub repositories. Many users report having
+    to piece together information from various sources.
+  - Lack of examples of production usage for both the SDKs and Collector.
+- Local debugging:
+  - It's difficult to know why exporting isn't working, and if the SDK is
+    dropping data. Better tooling for local debugging would help with
+    troubleshooting.
+- SDK configuration:
+  - Overall complexity (both configuration and concepts)
+  - SDK configuration is complex, and many users have pointed out that
+    troubleshooting the exporter is the most painful part.
+- Collector operations:
+  - Debugging guidance on the Collector configuration and its components is
+    needed. OTTL was mentioned a couple of times for being difficult to
+    troubleshoot.
+
+## Detailed Insights
+
+### General Information about Respondents
+
+The survey responses came from users, of whom the majority use OpenTelemetry in
+production (83%) and do not work for a vendor (77.4%). 82% consider their
+organization to be at an intermediate or advanced phase of their journey with
+observability. The platforms used leaned towards Kubernetes (68.2%) and AWS
+(34.6%), with bare metal still making up a respectable percentage (12%).
+Programming language usage was split between the usual suspects of Go, Java,
+JavaScript, Python and .NET, but had a wide spread and even languages without
+official API/SDK implementations, as shown in the second graph below. While
+those working on backends or operations were the majority of respondents, mobile
+and browser were represented as well, see the third graph:
```

Comment:
Perhaps these findings could be better presented as a list or as a sequence of short pagraphs.

Reply:
Hm, maybe @julianocosta89 would be up to create another custom chart :)

Reply:
We have the following charts on the survey already:

- Do you work for an Observability or APM (monitoring) vendor?
- Are you running OpenTelemetry in Production?
- Where in your Observability journey is your organization?
- Which platforms are you running OpenTelemety on?

I'll create a new one for this last questions (platforms), as the one generated by Google Forms is horrible, but the others seem good to go.

Regarding the programming languages, we already have a chart later on the article.

Reply:
WDYT?

![image](https://github.com/user-attachments/assets/f03ab009-3c4a-4630-9c29-ce9484cee045)